api/src/controllers/auth.ts

```diff
@@ -24,23 +24,21 @@ const router = Router();
 
 const authProviders = getAuthProviders();
 
-if (authProviders) {
-	for (const authProvider of authProviders) {
-		let authProviderRouter: Router | null = null;
+authProviders.forEach((authProvider) => {
```

Comment:
We rely on `for...of` all over the place in this codebase, so let's keep that consistent .

(the main reason is that it makes it easy to switch between async and non-async functions inside the loop. You can't "await" in a forEach callback, but you can in a for...of invocation)

Reply:
Boo! I hate that attitude. I think you should use whatever tool is best for the job, and I see  `forEach` as the more utilitarian function with `for` being reserved for special cases. But it's no biggie, and I'll keep it in mind.